### PR TITLE
🗑 #1173 Add Ability to remove user account on Twake

### PR DESCRIPTION
**Pull request by stephanevieira75**
## Remove user command
`docker-compose exec node ./bin/twake-cli users remove --id 62895cc6-8579-11eb-b52e-0242ac130003`

## Preview

https://user-images.githubusercontent.com/36481167/116423801-4c37d480-a841-11eb-9bd2-02ecaf4100ca.mp4

### CODE REVIEW

File: twake/backend/node/src/services/user/services/users/service.ts [hunk 1]
```diff
@@ -40,12 +41,14 @@ export class UserService implements UsersServiceAPI {
...
   }
 
-  save<SaveOptions>(
+  async save<SaveOptions>(
     item: User,
     options?: SaveOptions,
     context?: ExecutionContext,
   ): Promise<SaveResult<User>> {
-    throw new Error("Method not implemented.");
+    this.repository.save(item);
```
Comment (chamerling) [inline twake/backend/node/src/services/user/services/users/service.ts:49]: `await` it

File: twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts [hunk 2]
```diff
@@ -69,7 +73,7 @@ export const transformValueToDbString = (
     return "''"; //Not implemented yet
   }
   if (type === "string" || type === "json") {
-    if (type === "json") {
+    if (type === "json" && v !== null) {
```
Comment (RomaricMourgues) [inline twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts:76]: Why is it needed?
If you do you must at least also do it in mongo connector.

File: twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts [hunk 3]
```diff
@@ -38,6 +38,10 @@ export const transformValueToDbString = (
     return `${parseInt(v)}`;
   }
   if (type === "uuid" || type === "timeuuid") {
+    if (isNull(v)) {
+      return "null";
+    }
+
```
Comment (RomaricMourgues) [inline twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts:44]: Why is it needed?
If you do this kind of modification you must at least also do it in mongo connector.

Also, are we able to put "NULL" in a scylladb timeuuid / uuid column ?

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]

File: twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts (hunk 2 above)
Comment (stephanevieira75) [reply twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts:76]: Already done 😄

File: twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts (hunk 3 above)
Comment (stephanevieira75) [reply twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts:44]: Already done too 👍🏻

**Pull request merged**